### PR TITLE
Support for more than 1 column family in rockmq to store traceid

### PR DESCRIPTION
**Pull request by smellthemoon**
/kind improvement

### CODE REVIEW

**Comment by congqixia** [comment]
add milestone for offline notification
/cc @xiaofan-luan @czs007

File: internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go [hunk 1]
```diff
@@ -787,30 +798,45 @@ func (rmq *rocksmq) Consume(topicName string, groupName string, n int) ([]Consum
...
-			return nil, err
+		offset++
+
+		if iterPropersity.Valid() && string(iterPropersity.Key().Data()) == string(iter.Key().Data()) {
+			// the key of properties is the same with the key of payload
+			// to prevent mix message with or without propersity column family
+			propertiesValue = iterPropersity.Value().Data()
+			iterPropersity.Next()
+		} else {
+			// between 2.2.0 and 2.3.0, the key of Payload is topic/properties/msgid/Payload
```
Comment (xiaofan-luan) [inline internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go:827]: let's ignore the data for 2.3.0 beta.
Just make sure property empty is ok for 2.3

**Comment by xiaofan-luan** [comment]
/assign @yanliang567 
Can you verify the upgrade from milvus 2.2 to milvus 2.3 and milvus2.3beta to milvus 2.3?

**Comment by yanliang567** [comment]
@zhuwenxing @elstic are working on it.

**Pull request merged**